src/CodeGenHelpers/EnumValueBuilder.cs

```diff
@@ -14,12 +14,27 @@ internal EnumValueBuilder(string name, EnumBuilder builder, int? value)
             Enum = builder;
         }
 
+        public DocumentationComment XmlDoc { get; } = new DocumentationComment();
```

Comment:
this should probably be private and only exposed via the helper methods on the EnumValueBuilder.